src/Symfony/Component/HttpFoundation/JsonResponse.php

```diff
@@ -27,6 +27,10 @@ class JsonResponse extends Response
     protected $data;
     protected $callback;
 
+    // Encode <, >, ', &, and " for RFC4627-compliant JSON, which may also be embedded into HTML.
+    // 15 === JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_AMP | JSON_HEX_QUOT
+    private $encodingOptions = 15;
```

Comment:
can't we use the constants here instead of hardcoding the value ? These constants should always be available anyway